common/tools/dev-tool/src/util/admin/migrate-package/codemods/replaceSinonStubs.ts

```diff
@@ -11,6 +106,7 @@ import * as ts from "typescript"; // For using TypeScript factory methods
  * 3. sinon.stub(obj, "methodName").throwsException(someError) => vi.spyOn(obj, "methodName").mockRejectedValue(someError)
  * 4. sinon.restore() => vi.restoreAllMocks()
  * 5. sandbox.restore() => vi.restoreAllMocks()
+ * 6. sinon.stub() => vi.fn()
```

Comment:
Nice addition, thank you!

Reply:
I'm sure we are missing some but I think we have the majority now accounted for.